.travis.yml

```diff
@@ -8,10 +8,20 @@ go:
 install:
         - mkdir -p $HOME/gopath/src/unidoc/
         - ln -s $HOME/gopath/src/github.com/unidoc/unioffice $HOME/gopath/src/github.com/unidoc/unioffice
+        - mkdir -p $HOME/dotnet
+        - cd $HOME/gopath/src/github.com/unidoc/unioffice
+        - mv aspnetcore.tar.gz $HOME/dotnet
+        - cd $HOME/dotnet
+        - tar -xf aspnetcore.tar.gz
+        - export PATH=$PATH:$HOME/dotnet
+        - cd $HOME/gopath/src/github.com/unidoc/unioffice
+        - unzip validator.zip
+        - cd openxml-validator
+        - dotnet build
 
 script:
         - cd $HOME/gopath/src/github.com/unidoc/unioffice
-        - ./build-examples.sh
+        - ./validate.sh
```

Comment:
Let´s keep the build-examples.sh script separate as it is useful for users who want to build examples without validating.
Can put the `validate-examples.sh` into .ci/.travis folder

Reply:
fixed